src/v2/Apps/Collect/Routes/Collect/index.tsx

```diff
@@ -101,6 +102,10 @@ export const CollectApp: React.FC<CollectAppProps> = ({
         <Box>
           <ArtworkFilter
             viewer={viewer}
+            aggregations={
+              viewer.artworksConnection
+                .aggregations as SharedArtworkFilterContextProps["aggregations"]
```

Comment:
Another possiblity here is to do something like 
```
export type Aggregations = ArtworkFilterContextProps["aggregations"]
```
vs having to dig in via array syntax